src/catalog/src/builtin.rs

```diff
@@ -6198,17 +6198,18 @@ pub static MZ_SHOW_CLUSTER_REPLICAS: Lazy<BuiltinView> = Lazy::new(|| BuiltinVie
     mz_catalog.mz_clusters.name AS cluster,
     mz_catalog.mz_cluster_replicas.name AS replica,
     mz_catalog.mz_cluster_replicas.size AS size,
-    statuses.ready AS ready
+    COALESCE(statuses.ready, false) AS ready
 FROM
     mz_catalog.mz_cluster_replicas
         JOIN mz_catalog.mz_clusters
             ON mz_catalog.mz_cluster_replicas.cluster_id = mz_catalog.mz_clusters.id
-        JOIN
+        LEFT JOIN
             (
                 SELECT
                     replica_id,
-                    mz_unsafe.mz_all(status = 'ready') AS ready
-                FROM mz_internal.mz_cluster_replica_statuses
+                    min(hydrated) AS ready
```

Comment:
I find `bool_and` to be a bit clearer in situations like this!

```suggestion
                    bool_and(hydrated) AS ready
```